web/src/compose_ui.js

```diff
@@ -279,6 +280,10 @@ export function make_compose_box_full_size() {
 
     $("#compose").addClass("compose-fullscreen");
 
+    const viewport_height = message_viewport.height();
+    const navbar_sticky_height = $("#navbar-sticky-container").safeOuterHeight(true);
+    $("#compose #compose-container").css("height", viewport_height - navbar_sticky_height);
```

Comment:
There's something weird resulting in an extra (faded until hovered) X icon visible here when looking at the full screen compose box on top of Recent Conversations:

![image](https://github.com/zulip/zulip/assets/2746074/e07e209e-6416-4b4a-8e3a-b726584e449c)

Using inspect element, apparnetly it's the "close" button from recent topics showing through.